### PR TITLE
Fix:#3 Added the Comments in CSS file

### PR DESCRIPTION
**Pull request by SharkyBytes**
Hey I had added the comments and removed the commented code. Is it okay?

If you want that code to be remained please let me know :)

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -167,123 +178,170 @@ section {
...
+/* Styling for exchange item names */
 .exchange_name {
   padding: 0 10px;
   font-size: 15px;
 }
+
+/* Styling for exchange item amounts */
 .exchange_amount {
-  float: right;
+  float: right
```
Comment (akshayhallagade) [inline style.css:301]: semicolon missed

File: style.css [hunk 2]
```diff
@@ -1,96 +1,99 @@
...
   justify-content: center;
-  /* border: 1px solid black; */
 }
 
-.navbar_items {
-  float: left;
-}
-.logo img {
-  width: 80%;
-}
```
Comment (akshayhallagade) [inline style.css:42]: code got removed automatically in committed file.

**Review by akshayhallagade — CHANGES_REQUESTED** [review]
I appreciate the work you have done. 
sorry to say this but, just gotta change in plan. 
Actually, that would be better if we just create the comments section wise, so that it would be beneficial manage the whole code and will get to know which code is at what place where.
for example : 
1. comment for part 1 of code.
2. comment for part 2 of code.
3. comment for navbar things.

**Comment by GarvAgarwal26** [comment]
Ok Sir I am making the requested changes. Please do not assign this task to any new one. As I am new to open source it will be a great help from your side :)

**Comment by GarvAgarwal26** [comment]
Hey please check #7 ...maybe now your problem is solved :)

**Pull request closed without merging**